lib/pages/chat_list/space_view.dart

```diff
@@ -588,3 +802,15 @@ enum SpaceActions {
   invite,
   leave,
 }
+
+enum SpaceViewOptions {
+  normal(icon: Icons.sort),
+  roomTop(icon: Icons.vertical_align_top),
+  categorized(icon: Icons.category);
```

Comment:
This should be ...`category_underlined`for consistency.

Reply:
For all of them? Or just the category one?

Reply:
Yeah, all should be `_outlined`. That's just the only one I noticed, it was still pretty early when I did the code review.